src/components/button-minor/button-minor.spec.tsx

```diff
@@ -88,6 +89,19 @@ describe("Button Minor", () => {
     }
   );
 
+  it("when icon only with tooltip message, icon's position is absolute", () => {
+    const wrapper = mount(
+      <ButtonMinor iconType="bin" iconTooltipMessage="foo" />
```

Comment:
If you make the code change I suggested,the `iconTooltipMessage` prop won't be needed in this prop (and we can leave it out of the test description).